modules/rosa-sts-creating-a-cluster-quickly-cli.adoc

```diff
@@ -4,12 +4,12 @@
 // * rosa_getting_started/rosa-getting-started.adoc
 
 :_content-type: PROCEDURE
-[id="rosa-sts-creating-cluster-using-defaults{context}"]
-= Creating a ROSA cluster with STS using the default options
+[id="rosa-sts-creating-cluster-using-defaults-cli_{context}"]
+= Creating a cluster with the default options through the CLI
```

Comment:
@bergerhoffer thank you for your detailed review and for highlighting this. I have opted to update this to "using" instead of "by using" because it seems to fit well in these headings. I have removed the use of "through" in this context in the other headings in the PR too.